guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -208,8 +208,7 @@
         "ERROR_INVALID_CSV_HEADER": "Invalid CSV Header \"{HEADER}\" is neither an attribute or parameter",
         "ERROR_INVALID_MIME_TYPE": "Unsupported file type: \"{TYPE}\"",
         "ERROR_INVALID_GROUP": "No group matching \"{GROUP}\" found",
-        "ERROR_INVALID_USER_GROUP_IDENTIFIERS": "User Groups not found: {IDENTIFIER_LIST}",
```

Comment:
These are vestigal translations from an earlier version of the code would error out rather than creating unknown users and groups. They are not used anywhere in the app.